test/test-tcp-connect-timeout.c

```diff
@@ -39,7 +39,7 @@ static void close_cb(uv_handle_t* handle);
 
 static void connect_cb(uv_connect_t* req, int status) {
   ASSERT_PTR_EQ(req, &connect_req);
-  ASSERT_EQ(status, UV_ECANCELED);
+  ASSERT_EQ(status, 0);
```

Comment:
You probably already knew because of the CI failures but this change breaks the test.

If you're observing status==0 locally, that suggests something (firewall, iptables rule, whatever) causes a TCP connection to 8.8.8.8:9999 to succeed. It shouldn't, there's nothing listening there.